lib/ddtrace/contrib/active_storage/events/delete.rb

```diff
@@ -0,0 +1,59 @@
+require 'ddtrace/contrib/analytics'
+require 'ddtrace/contrib/active_storage/event'
+require 'ddtrace/ext/http'
+require 'ddtrace/contrib/active_storage/ext'
+
+module Datadog
+  module Contrib
+    module ActiveStorage
+      module Events
+        # Defines instrumentation for 'service_delete.active_storage' event.
+        # From: https://edgeguides.rubyonrails.org/active_support_instrumentation.html#active-storage
+        module Delete
```

Comment:
The events are not all the same, but I see quite a few similarities between them.
Would it be possible to create a shared component for these?
I understand that not all the fields are the same, and would require custom treatment.

Reply:
yes probably, i can take an approach similar to our kafka instrumentation